src/Visualization/Python/InterpolateVolumeData.py

```diff
@@ -82,14 +84,20 @@ def interpolate_h5_file(source_file_path,
         the name of the .vol file inside the target file where the target data
         is written. Requires leading `/` but no `.vol` file extension.
     obs_start: float, optional
-        disregards all observations with observation value before `obs_start`
+        disregards all observations with observation value before but not
+        including `obs_start`
     obs_end: float, optional
-        disregards all observations with observation value after `obs_end`
+        disregards all observations with observation value after but not
+        including `obs_end`
```

Comment:
I think you're currently _including_ both `obs_start` and `obs_end`

Reply:
technically that is what this sentence says ;) but I see the confusion comes from the sorta double negative.

Reply:
Oh right :D yea some rewording would help here :)